loadbalancer/zk/client.go

```diff
@@ -0,0 +1,82 @@
+package zk
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/samuel/go-zookeeper/zk"
+	"strings"
+	"time"
+)
+
+const ()
+
+// Client is a wrapper arround the zookeeper client.
+type Client interface {
+	// GetEntries queries the path in zookeeper, places a watch on it
+	// and retrieves data from its current child nodes
+	GetEntries(path string) ([]string, <-chan zk.Event, error)
+	// createParentNodes tries to create the path
+	// in case it does not exist yet on Zookeeper
+	createParentNodes(path string) error
+}
+
+type client struct {
+	*zk.Conn
+	evtConn <-chan zk.Event
```

Comment:
Old habit in naming logic dating from my Delphi days at a certain company I worked for. I'll rename.